geoportailv3/static/js/mymaps/mymaps.html

```diff
@@ -50,43 +50,82 @@
 </div>
 
 <!-- Modification Modal -->
-<ngeo-modal ngeo-modal-shown="ctrl.modifying">
+<ngeo-modal ngeo-modal-shown="ctrl.modifying || ctrl.creating">
```

Comment:
The expression passed to `ngeo-modal-shown` must be assignable. `ctrl.modifying` is assignable while `ctrl.modifying || ctrl.creating` is not.

The "assignable" property is important because the `ngeo-modal` directive changes the value of the "shown" property when the user closes the modal.